bigtable/test_bigtable.py

```diff
@@ -45,6 +50,29 @@ def tearDown(self):
         self.assert_command('gcloud bigtable instances delete {}'.format(
             self.db_name))
 
+    def buildParameters():
```

Comment:
What should we use as the default? 1.3?

Also, should the test take in only one image_version and then use the params with that given version? I'm asking because we currently run sets of tests containing different image versions.

If the answer is to take in several image_versions for testing more than one image_version, then which combinations should we be testing since there is no consistent format? 

For example, test_rstudio.py tests:
                ("SINGLE", "1.0", "rstudio", "password"),
                ("SINGLE", "1.1", "rstudio", "password"),
                ("SINGLE", "1.2", "rstudio", "password"),
                ("SINGLE", "1.3", "rstudio", "password"),
                ("SINGLE", "1.3", "", "password"),  # default username
                ("SINGLE", "1.3", "rstudio", ""),  # no auth
                ("SINGLE", "1.3", "", ""),  # default username and no auth
                ("SINGLE", "1.4", "rstudio", "password")

If we provided image_version=[1.3, 1.4] with params=[("SINGLE", "rstudio", "password"), ("SINGLE", "", password)], is it ok to test all the combinations? That is, [("SINGLE", "1.3", "rstudio", "password"), ("SINGLE", "1.3","", password), ("SINGLE", "1.4", "rstudio", "password"), ("SINGLE", "1.4","", password)].

Reply:
Yes, by default we can set it to `1.3`.

I think that we don't need even to have support for multiple versions in `image_version` flag, it will be simpler just to run separate build for each version track when necessary instead.

Reply:
Ok, cool.

Reply:
Done.